ios/Exponent/Versioned/Modules/Api/EXContacts.m

```diff
@@ -142,10 +227,125 @@ - (NSArray *)_emailsForContact:(CNContact * _Nonnull) person
       NSString *emailAddress = container.value;
       if (container.label) {
         NSString *emailLabel = [CNLabeledValue localizedStringForLabel:container.label];
-        [results addObject:@{ @"email": emailAddress, @"label": emailLabel }];
+        [results addObject:@{ @"email": emailAddress, @"label": emailLabel, @"id": container.identifier }];
+      } else {
+        [results addObject:@{ @"email": emailAddress, @"id": container.identifier }];
+      }
+    }
+  }
+  return results;
+}
+
+- (NSMutableDictionary *)_birthdayForContact:(NSDateComponents * _Nonnull) birthday
+{
+  NSMutableDictionary *result = nil;
+  if (birthday) {
```

Comment:
You specify `_Nonnull` in the method footprint, but you check again here, not sure which is intended